toolkit/javascripts/option-select.js

```diff
@@ -80,10 +80,13 @@
   };
 
   OptionSelect.prototype.checkedString = function checkedString(){
-    var count = this.$options.filter(":checked").size();
     var checkedString = "";
-    if (count > 0){
-      checkedString = count+" selected";
+
+    if (this.isClosed()) {
```

Comment:
Does this line of code tell to perform filtering automatically if filtering dialogue is closed?